README.md

```diff
@@ -162,7 +162,7 @@ Also note that we are only showing one projection below. The other projectors an
      │  │             │       │        └─────────────┘             │             │
      │  └─────────────┘   │  D. Read          │           │        └─────────────┘
      │         │              event      E. Update                        ▲
- A. Load  C. Save new     │   │          Projection       │               │
+ B. Load  C. Save new     │   │          Projection       │               │
```

Comment:
The order in these is correct. i.e. we load state into the aggregate before we ask it to handle the action e.g. https://github.com/envato/event_sourcery_todo_app/blob/master/app/commands/todo/add.rb#L45